syntax = "proto3";
package m2g.pb;

enum SpriteType {
  NO_SPRITE = 0;

  BELPER_EXPLOSIVE_PRESS = 1;
  BELPER_WOOD = 2;
  BELPER_JEWELLERY = 3;
  DERBY_EXPLOSIVE_KITCHEN = 4;
  DERBY_EXPLOSIVE_PRESS = 5;
  DERBY_TOOL = 6;
  LEEK_EXPLOSIVE_PRESS = 7;
  LEEK_EXPLOSIVE_WOOD = 8;
  STOKE_ON_TRENT_EXPLOSIVE_PRESS = 9;
  STOKE_ON_TRENT_JEWELLERY_TOOL = 10;
  STOKE_ON_TRENT_PRESS = 11;
  STONE_EXPLOSIVE_KITCHEN = 12;
  STONE_PRESS_WOOD = 13;
  UTTOXETER_PRESS_KITCHEN = 14;
  UTTOXETER_EXPLOSIVE_KITCHEN = 15;
  STAFFORD_PRESS_KITCHEN = 16;
  STAFFORD_JEWELLERY = 17;
  BURTON_UPON_TRENT_PRESS_WOOD = 18;
  BURTON_UPON_TRENT_KITCHEN = 19;
  CANNOCK_PRESS_WOOD = 20;
  CANNOCK_WOOD = 21;
  TAMWORTH_EXPLOSIVE_WOOD_1 = 22;
  TAMWORTH_EXPLOSIVE_WOOD_2 = 23;
  WALSALL_TOOL_PRESS = 24;
  WALSALL_PRESS_KITCHEN = 25;
  COALBROOKEDALE_TOOL_KITCHEN = 26;
  COALBROOKEDALE_TOOL = 27;
  COALBROOKEDALE_WOOD = 28;
  DUDLEY_WOOD = 29;
  DUDLEY_TOOL = 30;
  KIDDERMINSTER_EXPLOSIVE_WOOD = 31;
  KIDDERMINSTER_EXPLOSIVE = 32;
  WOLFERHAMPTON_PRESS = 33;
  WOLFERHAMPTON_PRESS_WOOD = 34;
  WORCESTER_EXPLOSIVE_1 = 35;
  WORCESTER_EXPLOSIVE_2 = 36;
  BIRMINGHAM_EXPLOSIVE_PRESS = 37;
  BIRMINGHAM_PRESS_1 = 38;
  BIRMINGHAM_PRESS_2 = 39;
  BIRMINGHAM_TOOL = 40;
  COVENTRY_JEWELLERY = 41;
  COVENTRY_PRESS_WOOD = 42;
  COVENTRY_TOOL_PRESS = 43;
  NUNEATON_PRESS_KITCHEN = 44;
  NUNEATON_EXPLOSIVE_WOOD = 45;
  REDDITCH_PRESS_WOOD = 46;
  REDDITCH_TOOL = 47;
  STANDALONE_KITCHEN_1 = 48;
  STANDALONE_KITCHEN_2 = 49;

  BELPER_DERBY_GUARD_KNIGHT = 51;
  BELPER_LEEK_KNIGHT = 52;
  DERBY_UTTOXETER_KNIGHT = 53;
  DERBY_BURTON_UPON_TRENT_GUARD_KNIGHT = 54;
  DERBY_NOTTINGHAM_GUARD_KNIGHT = 55;
  LEEK_STOKE_ON_TRENT_GUARD_KNIGHT = 56;
  STOKE_ON_TRENT_STONE_GUARD_KNIGHT = 57;
  STOKE_ON_TRENT_WARRINGTON_GUARD_KNIGHT = 58;
  STONE_UTTOXETER_KNIGHT = 59;
  STONE_STAFFORD_GUARD_KNIGHT = 60;
  STONE_BURTON_UPON_TRENT_GUARD_KNIGHT = 61;
  STAFFORD_CANNOCK_GUARD_KNIGHT = 62;
  BURTON_UPON_TRENT_CANNOCK_KNIGHT = 63;
  BURTON_UPON_TRENT_TAMWORTH_GUARD_KNIGHT = 64;
  BURTON_UPON_TRENT_WALSALL_GUARD = 65;
  CANNOCK_KITCHEN1_GUARD_KNIGHT = 66;
  CANNOCK_WALSALL_GUARD_KNIGHT = 67;
  CANNOCK_WOLFERHAMPTON_GUARD_KNIGHT = 68;
  TAMWORTH_WALSALL_KNIGHT = 69;
  TAMWORTH_BIRMINGHAM_GUARD_KNIGHT = 70;
  TAMWORTH_NUNEATON_GUARD_KNIGHT = 71;
  WALSALL_WOLFERHAMPTON_GUARD_KNIGHT = 72;
  WALSALL_BIRMINGHAM_GUARD_KNIGHT = 73;
  COALBROOKEDALE_KIDDERMINSTER_GUARD_KNIGHT = 74;
  COALBROOKEDALE_WOLFERHAMPTON_GUARD_KNIGHT = 75;
  COALBROOKEDALE_SHREWSBURY_GUARD_KNIGHT = 76;
  DUDLEY_KIDDERMINSTER_GUARD_KNIGHT = 77;
  DUDLEY_WOLFERHAMPTON_GUARD_KNIGHT = 78;
  DUDLEY_BIRMINGHAM_GUARD_KNIGHT = 79;
  KIDDERMINSTER_WORCESTER_KITCHEN2_GUARD_KNIGHT = 80;
  WORCESTER_BIRMINGHAM_GUARD_KNIGHT = 81;
  WORCESTER_GLOUCESTER_GUARD_KNIGHT = 82;
  BIRMINGHAM_COVENTRY_GUARD_KNIGHT = 83;
  BIRMINGHAM_NUNEATON_KNIGHT = 84;
  BIRMINGHAM_REDDITCH_KNIGHT = 85;
  BIRMINGHAM_OXFORD_GUARD_KNIGHT = 86;
  COVENTRY_NUNEATON_KNIGHT = 87;
  REDDITCH_GLOUCESTER_GUARD_KNIGHT = 88;
  REDDITCH_OXFORD_GUARD_KNIGHT = 89;

  GLOUCESTER_1 = 90;
  GLOUCESTER_2 = 91;
  SHREWSBURY_1 = 92;
  OXFORD_1 = 93;
  OXFORD_2 = 94;
  NOTTINGHAM_1 = 95;
  NOTTINGHAM_2 = 96;
  WARRINGTON_1 = 97;
  WARRINGTON_2 = 98;

  GRASS_1 = 1001;
  GRASS_2 = 1011;
  GRASS_3 = 1021;
  GRASS_4 = 1031;

  CANAL_VERTICAL_1 = 1041;
  CANAL_VERTICAL_2 = 1042;

  CANAL_HORIZONTAL_1 = 1051;
  CANAL_HORIZONTAL_2 = 1052;

  CANAL_TOP_RIGHT_TO_BOTTOM_LEFT_1 = 1061;
  CANAL_TOP_RIGHT_TO_BOTTOM_LEFT_2 = 1062;
  CANAL_TOP_RIGHT_TO_BOTTOM_LEFT_3 = 1063;
  CANAL_TOP_RIGHT_TO_BOTTOM_LEFT_4 = 1064;

  CANAL_TOP_LEFT_TO_BOTTOM_RIGHT_1 = 1071;
  CANAL_TOP_LEFT_TO_BOTTOM_RIGHT_2 = 1072;
  CANAL_TOP_LEFT_TO_BOTTOM_RIGHT_3 = 1073;
  CANAL_TOP_LEFT_TO_BOTTOM_RIGHT_4 = 1074;

  RAILWAY_VERTICAL = 1081;

  RAILWAY_HORIZONTAL = 1091;

  RAILWAY_TOP_RIGHT_TO_BOTTOM_LEFT_1 = 1101;
  RAILWAY_TOP_RIGHT_TO_BOTTOM_LEFT_2 = 1102;
  RAILWAY_TOP_RIGHT_TO_BOTTOM_LEFT_3 = 1103;
  RAILWAY_TOP_RIGHT_TO_BOTTOM_LEFT_4 = 1104;

  RAILWAY_TOP_LEFT_TO_BOTTOM_RIGHT_1 = 1111;
  RAILWAY_TOP_LEFT_TO_BOTTOM_RIGHT_2 = 1112;
  RAILWAY_TOP_LEFT_TO_BOTTOM_RIGHT_3 = 1113;
  RAILWAY_TOP_LEFT_TO_BOTTOM_RIGHT_4 = 1114;

  CANAL_AND_RAILWAY_TOP_RIGHT_TO_BOTTOM_LEFT_1 = 1121;
  CANAL_AND_RAILWAY_TOP_RIGHT_TO_BOTTOM_LEFT_2 = 1122;
  CANAL_AND_RAILWAY_TOP_RIGHT_TO_BOTTOM_LEFT_3 = 1123;
  CANAL_AND_RAILWAY_TOP_RIGHT_TO_BOTTOM_LEFT_4 = 1124;

  CANAL_AND_RAILWAY_TOP_LEFT_TO_BOTTOM_RIGHT_1 = 1131;
  CANAL_AND_RAILWAY_TOP_LEFT_TO_BOTTOM_RIGHT_2 = 1132;
  CANAL_AND_RAILWAY_TOP_LEFT_TO_BOTTOM_RIGHT_3 = 1133;
  CANAL_AND_RAILWAY_TOP_LEFT_TO_BOTTOM_RIGHT_4 = 1134;

  INDUSTRY_BACKGROUND = 1141;

  CANAL_OR_RAILWAY_BACKGROUND_1 = 1151;
  CANAL_OR_RAILWAY_BACKGROUND_2 = 1152;
  CANAL_OR_RAILWAY_BACKGROUND_3 = 1153;

  TEXT_LABEL_WARRINGTON = 2001;
  TEXT_LABEL_STOKE_ON_TRENT = 2002;
  TEXT_LABEL_LEEK = 2003;
  TEXT_LABEL_BELPER = 2004;
  TEXT_LABEL_STONE = 2005;
  TEXT_LABEL_UTTOXETER = 2006;
  TEXT_LABEL_DERBY = 2007;
  TEXT_LABEL_NOTTINGHAM = 2008;
  TEXT_LABEL_STAFFORD = 2009;
  TEXT_LABEL_BURTON_UPON_TRENT = 2010;
  TEXT_LABEL_CANNOCK = 2011;
  TEXT_LABEL_TAMWORTH = 2012;
  TEXT_LABEL_SHREWSBURY = 2013;
  TEXT_LABEL_COALBROOKEDALE = 2014;
  TEXT_LABEL_WOLFERHAMPTON = 2015;
  TEXT_LABEL_WALSALL = 2016;
  TEXT_LABEL_NUNEATON = 2017;
  TEXT_LABEL_KIDDERMINSTER = 2018;
  TEXT_LABEL_DUDLEY = 2019;
  TEXT_LABEL_BIRMINGHAM = 2020;
  TEXT_LABEL_COVENTRY = 2021;
  TEXT_LABEL_WORCESTER = 2022;
  TEXT_LABEL_MAASTRICHT = 2023;
  TEXT_LABEL_OXFORD = 2024;
  TEXT_LABEL_GLOUCESTER = 2025;
}
